tests/playwright/pageobjects/page.js

```diff
@@ -0,0 +1,23 @@
+const querystring = require( 'querystring' );
+
+class Page {
+	/**
+	 * Navigate the browser to a given page.
+	 *
+	 * @param {string} title Page title
+	 * @param {Object} [query] Query parameter
+	 * @param {string} [fragment] Fragment parameter
+	 * @return {void} This method runs a browser command.
+	 */
+	async openTitle( title, query = {}, fragment = '' ) {
+		query.title = title;
+		await page.goto(
```

Comment:
why is it that in some places we use `global.page` while at others we use just `page`